javacorelib/src/main/scala/java/lang/System.scala

```diff
@@ -63,21 +63,18 @@ object System {
       }
       sysProps.setProperty("user.home", getenv("HOME"))
       val buf = stackalloc[scala.Byte](1024)
-      unistd.getcwd(buf, 1024) match {
-        case null =>
-        case b    => sysProps.setProperty("user.dir", fromCString(b))
-      }
+      sysProps.setProperty("user.dir", null)
     }
 
     sysProps
   }
 
   var in: InputStream =
-    new FileInputStream(FileDescriptor.in)
+    null //new FileInputStream(FileDescriptor.in)
   var out: PrintStream =
-    new PrintStream(new FileOutputStream(FileDescriptor.out))
+    null //new PrintStream(new FileOutputStream(FileDescriptor.out))
   var err: PrintStream =
-    new PrintStream(new FileOutputStream(FileDescriptor.err))
+    null //new PrintStream(new FileOutputStream(FileDescriptor.err))
```

Comment:
Before we started using `FileInputStream` and `FileOutputStream`, we used to have a dedicated private stream implementations that didn't rely on File I/O: https://github.com/scala-native/scala-native/commit/8041de7f11a49e9922883770f01e57c454a5358e#diff-9f67b7df156c5096453ed90792be0ad2

You can just revert that change here. This will retain the in/out/err working without having `javacorelib` depend on `javafilelib`.

Reply:
Thanks, this is only part of the revert, FileDescriptor.out also should be reverted then, I used `std::cout`, `std::cerr` and `std::cin` implementation in my another Windows support branch, so I can port only these lines and then in another PR I will submit whole implementation of FileDescriptor I have already. How does it sounds?

Reply:
Should work now